src/trailing-slashes.txt

```diff
@@ -1,24 +1,32 @@
+/aidrop-nitro
 /airdorp
 /airdrop
 /billing
+/claim
+/claim-nitro
 /discordnitro
 /drop
+/free-gift
 /free-nitro
+/freenitro
 /get
 /gift
-/gift-nitro
 /gift_nitro
+/gift-nitro
 /glft
 /joinchat
+/newyear/freenitro
 /nitro
 /nitro-accept
 /nitro-gift
+/nitro-promo
 /nitrodiscord
 /nitrodrop
 /nitrogift
 /nitrogive
-/newyear/freenitro
 /promo
+/promo
+/promotion
```

Comment:
These two seem way to broad - there are many websites which use these

Reply:
what does broad mean?

Reply:
I mean that a lot of legitimate websites who have promotions or promotional offers will be broken.
Also, the rule `/promo` seems redundant anyway

Reply:
Yeah, those could cause false-positives.